ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml

```diff
@@ -44,6 +44,21 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: installer-rehearse-gcp-upi
+  interval: 960h
+  steps:
+    cluster_profile: gcp
+    workflow: cucushift-installer-rehearse-gcp-upi
+- as: installer-rehearse-gcp-upi-xpn
```

Comment:
if  `cluster_profile` did not change, we can reuse the existing test grid to test the new workflow, instead of introduce new test grid, just replace the 46 line with the new workflow.

Reply:
As the changes to "upi-install-gcp-commands.sh" may possibly affect basic GCP UPI workflow, I keep it there.